### PR TITLE
Focus Android 7.0 strings

### PR DESCRIPTION
**Pull request by boek**

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -452,6 +469,21 @@ msgctxt "preference_privacy_should_block_cookies_yes_option"
...
+msgctxt "preference_security_biometric"
+msgid "Use fingerprint to unlock app"
+msgstr ""
+
+#. Preference Summary: Biometrics (iris scan, fingerprint, or facial scan) can
+#. be used to unlock Focus if the app had a URL         open when the app was
+#. last run to ensure that only the phone's owner can view a previous session.
+msgctxt "preference_security_biometric_summary"
+msgid ""
+"Your fingerprint can unlock Firefox Focus if a URL is already opened in "
```
Comment (Delphine) [inline locales/templates/app.pot]: "Firefox Focus" -> I think this should be "%1$s" (+ add comment explaining it refers to app)

File: locales/templates/app.pot [hunk 2]
```diff
@@ -466,7 +498,11 @@ msgstr ""
 #. other displays. However we try to keep this description short and brief to
```
Comment (Delphine) [inline locales/templates/app.pot]: Nit: I guess it's not short and brief anymore? ^^ i'll let you decide if it's worth taking that off the localization note or not

File: locales/templates/app.pot [hunk 3]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+
+#. Title for the prompt to enable search suggestions
+msgctxt "enable_search_suggestion_title"
+msgid "Would you like to turn on search suggestions?"
+msgstr ""
+
+#. Description for the prompt to enable search suggestions          %1$s will
+#. be replaced by the app name (e.g. Firefox Focus)           %2$s will be
+#. replaced with the "Learn more" string with the name:
+#. enable_search_suggestion_subtitle_learnmore
```
Comment (Delphine) [inline locales/templates/app.pot]: I think the end of the sentence needs to be cut off sooner? Like "%2$s will be
replaced with "Learn more" - something like that. Or am I missing something?
Side question: why would "learn more" be a placeholder here? I really don't know TBH

File: locales/templates/app.pot [hunk 4]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+#, c-format
+msgctxt "enable_search_suggestion_subtitle"
+msgid ""
+"To get search suggestions, %1$s needs to send what you type in the "
+"address bar to the search engine you've chosen. %2$s."
+msgstr ""
+
+#. Learn more string to be placed inside: enable_search_suggestion_subtitle
```
Comment (Delphine) [inline locales/templates/app.pot]: Nit: this seems confusing to me, could we add quotes around "Learn more" please? And explain what happens here more explicitly? Seems more like a dev comment than a localization comment/explanation for what's going on :)

File: locales/templates/app.pot (hunk 3 above)
Comment (Delphine) [reply locales/templates/app.pot]: Ah looking at the next string I think I understand why it's a placeholder. But I find the explanation confusing nonetheless :)

File: locales/templates/app.pot [hunk 5]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+msgctxt "enable_search_suggestion_subtitle_learnmore"
+msgid "Learn more"
+msgstr ""
+
+#. Label used for the no button for the prompt to enable search suggestions
```
Comment (Delphine) [inline locales/templates/app.pot]: Same thing please put quotes around "no button"

File: locales/templates/app.pot [hunk 6]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+msgctxt "enable_search_suggestions_no"
+msgid "No"
+msgstr ""
+
+#. Label used for the yes button for the prompt to enable search suggestions
```
Comment (Delphine) [inline locales/templates/app.pot]: "yes button" (add quotes)

File: locales/templates/app.pot [hunk 7]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+
+#. Button to dismiss the message displayed  when using a search engine that
+#. doesn't provide suggestions
+msgctxt "dismiss_no_suggestions_prompt_button"
+msgid "Dismiss"
+msgstr ""
+
+#. Tips displayed on the home view
+#. Tip displayed on home view explaining how to disable tracking protection
+#. The newline here must be kept as the second half of the string is clickable
```
Comment (Delphine) [inline locales/templates/app.pot]: Nits: 
* the "newline" -> "new line"
* Also, beginning of sentence repeating twice (you can take out the first part)

File: locales/templates/app.pot [hunk 8]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+" Try turning off Tracking Protection"
+msgstr ""
+
+#. Tip displayed on home view explaining how to add sites to homescreen
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_add_to_homescreen"
+msgid ""
+"Get one-tap access to sites you use most\n"
+" Menu &gt; Add to Home screen"
```
Comment (Delphine) [inline locales/templates/app.pot]: I think the tag should be converted? Not sure. We use that in Firefox / Fennec strings, but haven't seen it in other products

File: locales/templates/app.pot [hunk 9]
```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
...
+" Long-press any link on a page"
+msgstr ""
+
+#. Tip displayed on home view explaining how to request the desktop version of
+#. a website.      The newline here must be kept as the second half of the
+#. string is clickable for the user.
+msgctxt "tip_request_desktop"
+msgid ""
+"Rather see the full desktop site?\n"
+" Menu &gt; Request desktop site"
```
Comment (Delphine) [inline locales/templates/app.pot]: Again not sure if the entity should be replaced here by the actual html tag. I can check about that

**Review by Delphine — CHANGES_REQUESTED** [review]
Ok started a review, I can check again against a fresh commit once these have been addressed :)
thanks! (and no rush!)

File: locales/templates/app.pot (hunk 3 above)
Comment (boek) [reply locales/templates/app.pot]: ah, the line breaks are weird here for sure!

Here is an updated description, does this make more sense?
```
Description for the prompt to enable search suggestions
        %1$s will be replaced by the app name (e.g. Firefox Focus)
        %2$s will be replaced with "Learn more" which will be a tappable link
        using the string: "enable_search_suggestion_subtitle_learnmore"
```

File: locales/templates/app.pot (hunk 8 above)
Comment (boek) [reply locales/templates/app.pot]: Hmm the string in the source code is `>`. Not sure why it's getting converted here
```
<string name="tip_add_to_homescreen">Get one-tap access to sites you use most\n
        Menu > Add to Home screen</string>
```

**Comment by boek** [comment]
closing in favor of #39

**Pull request closed without merging**